### PR TITLE
[Travis] update node and firefox config

### PR DESCRIPTION
**Pull request by dfreedm**
Set Node version to v6 LTS
Use latest firefox
Remove sauce_addon plugin, WCT will handle it

### CODE REVIEW

**Comment by dfreedm** [comment]
@rictic PTAL

**Comment by dfreedm** [comment]
paper-checkbox is a bit special right now, so I've excluded it from the travis pass

File: config.json [hunk 1]
```diff
@@ -69,7 +69,8 @@
     },
     "travis": {
       "blacklist": [
-        "repos/polymer-starter-kit"
+        "repos/polymer-starter-kit",
+        "repos/paper-checkbox"
```
Comment (rictic) [inline config.json:75]: why blacklist paper-checkbox?

File: cleanup-passes/travis.ts [hunk 2]
```diff
@@ -88,12 +88,12 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
   const ta = travis.addons;
 
   // use latest firefox, unless specified
-  if (!ta.firefox) {
-    ta.firefox = 'latest';
-  }
+  ta.firefox = 'latest';
```
Comment (rictic) [inline cleanup-passes/travis.ts:91]: This looks like it always overwrites, which isn't what the comment says. Update the comment?

File: config.json (hunk 1 above)
Comment (dfreedm) [reply config.json:75]: paper-checkbox is a bit special right now (eslint example element), so I've excluded it from the travis pass

Comment (rictic) [reply config.json:75]: k. This json file is preprocessed so it can have comments. add one here?

File: cleanup-passes/travis.ts (hunk 2 above)
Comment (dfreedm) [reply cleanup-passes/travis.ts:91]: Done

File: config.json (hunk 1 above)
Comment (abdonrd) [reply config.json:75]: @azakus now I'm trying this approach to use ESLint in elements: [IBMResearch/ibm-tabs/master/.travis.yml#L22-L23](https://github.com/IBMResearch/ibm-tabs/blob/master/.travis.yml#L22-L23). Maybe can help you.

Comment (dfreedm) [reply config.json:75]: @abdonrd You could just commit the eslintrc.json file to your repo, but otherwise seems reasonable.

Comment (dfreedm) [reply config.json:75]: @rictic Done

Comment (abdonrd) [reply config.json:75]: @azakus oops! Right! I had done that because I had a problem with the editor. Updated! Thanks!

**Review by rictic — APPROVED** [review]

**Pull request merged**